ynr/apps/elections/uk/templates/candidates/person-view.html

```diff
@@ -142,7 +142,13 @@ <h2>Candidacies:</h2>
         {% else %}
            Contesting the {{ election_data.name }} ({{ election_data.election_date }})
         {% endif %}</dt>
+       
       <dd>{{ person|post_in_election:election_data }}</dd>
+      {% for membership in person.memberships.all %}
```

Comment:
I think this is looping through all of the memberships for every election rather than 1-1 matching a membership to an election - check out my comment for more info :)

Reply:
Thanks for catching this; I've updated the tests and template to address this.